app/src/main/java/app/inf1d_team3/open_day/OpenDayActivity.java

```diff
@@ -56,4 +56,16 @@ public void addToCalendar(View view){
                 .putExtra(CalendarContract.Events.AVAILABILITY, CalendarContract.Events.AVAILABILITY_TENTATIVE);
         startActivity(intent);
     }
+
+    public void share(View view){
+        Intent myIntent = new Intent(Intent.ACTION_SEND);
+        myIntent.setType("text/plain");
+        String shareSub= getResources().getString(R.string.open_day_shareTitle) + " " + openDay.name;
+        String shareBody=  shareSub + "\n\n"
+                + openDay.description + "\n\n"
+                + openDay.location;
+        myIntent.putExtra(Intent.EXTRA_SUBJECT,shareSub);
+        myIntent.putExtra(Intent.EXTRA_TEXT,shareBody);
+        startActivity(Intent.createChooser(myIntent, "share using"));
```

Comment:
The share menu still says "share using"
We forgot to implement this part, this gets the correct string from res/values/strings.xml
```suggestion
        startActivity(Intent.createChooser(myIntent, getResources().getString(R.string.open_day_shareMenuName)));
```